packages/react-dom-bindings/src/client/ReactDOMHostConfig.js

```diff
@@ -1389,3 +1564,22 @@ export {
   acquireResource,
   releaseResource,
 } from './ReactDOMFloatClient';
+
+// -------------------
+//     Singletons
+// -------------------
+
+export const supportsSingletons = true;
+
+export function isHostSingletonInstance(
```

Comment:
This doesn't need to be exported, does it?

Reply:
function is gone now